pkgs/by-name/va/vault-tasks/package.nix

```
@@ -2,20 +2,23 @@
   lib,
   fetchFromGitHub,
   rustPlatform,
+  nix-update-script,
 }:
 rustPlatform.buildRustPackage rec {
```

nit: `rec` is considered bad practice per https://nix.dev/guides/best-practices.html.

```suggestion
let 
    version = "0.8.0";
in
rustPlatform.buildRustPackage {
    inherit version;
```